### DISCUSSION TITLE
安卓手机版更新最新版也不能用

### DISCUSSION THREAD

**Discussion by tytechan** [getlantern/lantern-forum-cn#507] [General]
如题，打开后显示连上洛杉矶或新加坡都没法用。手机用的不多，也试过不少版本了，都不行，有解决方案吗？

**Comment by MaxMKMax** [comment]
@tytechan 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

1. go to Support (转到支持)
2. select Report an Issue (选择报告问题)
3. choose issue type (选择问题类型)
4. include additional details (包括额外的详细信息)
5. hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by tytechan** [reply]
72405643

**Reply by MaxMKMax** [reply]
@tytechan 好的. 谢谢您的log. 我会帮您提交上去. 同时您试看重新卸载和下载可以解决问题吗.

**Reply by tytechan** [reply]
重新卸载重装也不行。

**Reply by tytechan** [reply]
还没有结果吗?

**Reply by MaxMKMax** [reply]
@tytechan 您好. 其他安卓手机的用户没有这个问题. 好像只有您遇到这个问题. 请问您使用哪一款安卓呢? 安卓哪个版本? 这个情况很奇怪. 是不是您的手机上面的设定有问题? 还是关掉权限了? 您帮忙查看一下. 谢谢!

**Reply by liuys710** [reply]
一样的，安卓手机装上用不了，显示连接蓝灯，但是什么网站都打不开

**Reply by tytechan** [reply]
魅族18，flyme 10.5.0.0A。所以这个问题就这样遗留了？

**Comment by MaxMKMax** [comment]
@tytechan https://lantern.io/zh-Hans/faq
您看一下这个能帮到你吗?

**Reply by tytechan** [reply]
帮不到

**Comment by MaxMKMax** [comment]
@tytechan 
您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)
当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by tytechan** [reply]
怎么又要提交？
